src/main/java/TUM.java

```diff
@@ -0,0 +1,54 @@
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+
+public class TUM {
+    public static void greetToUsers() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm TUM");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+    public static void main(String[] args) {
+        List<Task> taskList = new LinkedList<>();
+        greetToUsers();
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                System.out.println("____________________________________________________________");
+                System.out.println("     Here are the tasks in your list:");
+                for(int i = 0; i < taskList.size(); i++) {
+                    System.out.println("     " + (i+1) + ".[" + taskList.get(i).getStatusIcon() + "] " + taskList.get(i).getDescription());
```

Comment:
Line #23 is quite long. Maybe you should break it up into separate line.
```suggestion
                    System.out.println("     " + (i+1) + ".[" + taskList.get(i).getStatusIcon() + "] " 
                    + taskList.get(i).getDescription());
```